CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
+
+Only the Committers have write access to the repository and they are voted for in elections monitored by the Eclipse Foundation staff.
 
 ### Change Requests [2.i]
 
-The source code repository for `CycloneDDS` is hosted in GitHub, and the change process follows a typical GitHub pull request model.
-There are several [team members](https://projects.eclipse.org/projects/iot.cyclonedds/who) with rights to merge the pull requests, and an election process exists for promoting new members of that team.
+All changes will occur through a pull request, check ROS 2 Developer Guide for additional information.
 
 ### Contributor Origin [2.ii]
 
-Perspective contributors to `CycloneDDS` must accept the terms of the [Eclipse Contributor Agreement](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement).
-This is confirmed by a signed-off-by line in the commits put forth for review, which is enforced by a GitHub plugin which blocks completion of the pull request until all contributions contain a sign-off.
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](https://www.eclipse.org/legal/DCO.php).
 
 ### Peer Review Policy [2.iii]
 
-There is no published policy for peer review of `CycloneDDS` contributions, but because all changes are made through a pull request, each requires an authorized committer to be merged.
+All pull requests will be peer-reviewed, check [Eclipse Developer Process](https://www.eclipse.org/projects/dev_process/) for additional information.
 
 ### Continuous Integration [2.iv]
 
+All pull requests must pass CI on all tier 1 platforms
+Currently nightly results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/rmw_cyclonedds_cpp/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/rmw_cyclonedds_cpp/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/rmw_cyclonedds_cpp/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/rmw_cyclonedds_cpp/)
+
 Incoming pull requests automatically trigger continuous integration testing and must run successfully to be merged.
 Automated testing runs for:
 - Ubuntu Xenial with gcc 8 and Clang 7.0.0
 - macOS Mojave with Xcode 11.1
 - macOS Sierra with Xcode 9
 - Windows Server semi-annual release with Visual Studio 2017
 
+### Documentation Policy [2.v]
+All pull requests must resolve related documentation changes before merging.
+
 ## Documentation [3]
 
 ### Feature Documentation [3.i]
 
 [Project documentation](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/docs/dev/modules.md#feature-discovery) states that the features available in the product are "largely dynamic" based on compile-time discovery.
```

Comment:
I would say this is a mischaracterization of the situation. There are few things that can be disabled at compile time, either
* explicitly (e.g., DEADLINE support)
* because some prerequisites are missing (e.g., the security plugins depend on OpenSSL, and in the absence of it, those won't be built)
* platform dependent features auto-discovered at build-time (e.g., IPv6 support, availability of DNS — more of an embedded thing than for the tier platforms of ROS)

In the context of ROS 2 builds, none of those are an issue and all features are available always.